src/pages/support/faq.md

```diff
@@ -26,13 +27,35 @@ When retrieving data from some external systems your code might need to connect
 
 If you have a need for a port that is not in this list, please share with us the use case.
 
+## How can I see usgae reports for my Runtime namespace?
+Runtime currently does not offer usage reports but it is planned to be available in the future. To obtain usage reports for your namespace, kindly reach out to your account manager.
+
+To request usage reports for your namespace, please follow these steps:
+1. Contact your account manager and request for the usage report.
+2. Provide your namespace details to the account manager.
+3. The account manager will provide you with the usage report once it becomes available.
+
+Note: The availability of usage reports is currently not offered by Runtime but it is in the product roadmap. We appreciate your patience and understanding as we continue to improve our services.
+
+## How can I get access to the I/O Runtime activation logs?
+I/O Runtime doesn’t offer a configuration to send activations and logs to an external system, something like Splunk, Datadog or New Relic. This is something we are considering to offer in the future.
```

Comment:
Why does the documentation state that IO Runtime doesn't provide a possibility to send logs to 3rd party systems if we already provide log forwarding? It looks confusing.

Reply:
Also, using `activations` here looks confusing as well.